cmd/vicadmin/vicadm.go

```diff
@@ -51,14 +51,19 @@ import (
 var (
 	logFileDir = "/var/log/vic/"
 
+	// constants for any kind of username/password based authentication
+	usernameEnvVar = "vicadmin_username"
```

Comment:
Looks like this and the one below don't change.  If they're indeed constants, can we make them `const`?